syntax = "proto3";

package terragen;

message PositionUpdateBroadcastModel {
  string playerName = 1;
  float x = 2;
  float y = 3;
  float z = 4;
  float rotationX = 5;
}